pkg/await/pod.go

```diff
@@ -197,6 +197,10 @@ func (pia *podInitAwaiter) Read() error {
 }
 
 func (pia *podInitAwaiter) processPodEvent(event watch.Event) {
+	if event.Object == nil {
+		glog.V(3).Infof("received event with nil Object: %#v", event)
```

Comment:
Under what circumstances does this occur?

Reply:
In the linked issue, it seemed like it was caused by a network interruption.